.azure/templates/steps/system_test_general.yaml

```diff
@@ -107,7 +107,15 @@ jobs:
         publishJUnitResults: true
         testResultsFiles: '**/failsafe-reports/TEST-*.xml'
         goals: 'verify'
-        options: '-P${{ parameters.profile }} -DexcludedGroups=flaky,loadbalancer,networkpolicies,${{ parameters.excludedGroups }} -Dmaven.javadoc.skip=true -B -V -Dfailsafe.rerunFailingTestsCount=2 -Djunit.jupiter.execution.parallel.enabled=${{ parameters.run_parallel }} -Djunit.jupiter.execution.parallel.config.fixed.parallelism="${{ parameters.parallel }}"'
+        options: '
+          -P${{ parameters.profile }} 
+          -DexcludedGroups=flaky,loadbalancer,networkpolicies,${{ parameters.excludedGroups }} 
+          -Dmaven.javadoc.skip=true -B -V 
```

Comment:
Should `-B` and `-V` be on a separate line as well?

Reply:
Correct